<?xml version="1.0"?>
<launch>
    
    <!--Arguments-->
    <arg name="r_name" default="racecar1"/>

    <param name="r_name" value="racecar"/>

    <!--Parameters-->
    <rosparam file="$(find racecar_clear_ev_route)/config/single_agent_qlearning_master_config.yaml" command="load" ns="/$(arg r_name)"/>
    <rosparam file="$(find racecar_rl_environments)/config/racecar_clear_ev_route_basic_env_config.yaml" command="load"/>


    <node name="clear_ev_route_basic_env" pkg="racecar_rl_environments" type="racecar_clear_ev_route_basic_env.py" output="screen">
    </node>

</launch>
